cli/src/tests.rs

```diff
@@ -558,6 +558,43 @@ async fn create_workspace(tmp_path: TmpPath) {
         ));
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn list_users(tmp_path: TmpPath) {
+    let tmp_path_str = tmp_path.to_str().unwrap();
+    let config = get_testenv_config();
+    let (url, [alice, ..], _) = run_local_organization(&tmp_path, None, config)
+        .await
+        .unwrap();
+
+    set_env(&tmp_path_str, &url);
+
+    Command::cargo_bin("parsec_cli")
+        .unwrap()
+        .args(["list-users", "--device", &alice.slughash()])
+        .assert()
+        .stdout(predicates::str::contains(format!("No users found",)));
+
+    Command::cargo_bin("parsec_cli")
+        .unwrap()
+        .args([
+            "create-workspace",
+            "--device",
+            &alice.slughash(),
+            "--name",
+            "new-workspace",
+        ])
+        .unwrap();
```

Comment:
Why creating a workspace _magically_ added 3 users ?

Reply:
I have to investigate I guess

Reply:
I found that a `sync` was missing, and the `create_workspace` `sync` it